community/index.html

```diff
@@ -10,11 +10,11 @@
     <div class="row">
         <div class="col-md-11 nofloat center-block">
             <div class="community col-lg-9 col-md-10 col-sm-12">
-                <p class="lead">gRPC has an active community of developers who are using, enhancing and building valuable integrations with other software projects. We’d love your help to improve and extend the project. You can reach us via mail, IRC chat, twitter, gitter channel</p>
+                <p class="lead">gRPC has an active community of developers who are using, enhancing and building valuable integrations with other software projects. We’d love your help to improve and extend the project. You can reach us via mail, IRC chat, twitter, gitter channel to start engaging with the project and its members. </p>
```

Comment:
Could you please wrap this line and also add links to twitter and gitter when you mention this here. 
As for IRC we should mention that it's on Freenode,  #grpc